e2e-tests/mdx/gatsby-node.js

```diff
@@ -15,17 +15,4 @@ exports.onPostBuild = async ({ graphql }) => {
     results,
     { spaces: 2 }
   )
-}
-
-exports.createSchemaCustomization = ({ actions }) => {
-  const { createTypes } = actions
-
-  createTypes(`
-    type Mdx implements Node {
-      frontmatter: Frontmatter
-    }
-    type Frontmatter {
-      title: String
-    }
-  `)
-}
+}
```

Comment:
Why did you remove that?

Reply:
Should be not needed? The plugin now takes care of the schema?

Reply:
ah, ok 👍